src/ctia/entity/asset.clj

```diff
@@ -0,0 +1,135 @@
+(ns ctia.entity.asset
+  (:require [ctia.domain.entities :refer [default-realize-fn]]
+            [ctia.schemas.core :refer [def-acl-schema def-stored-schema]]
+            [ctia.schemas.utils :as csu]
+            [ctia.schemas.sorting :as sorting]
+            [ctia.stores.es.mapping :as em]
+            [ctia.stores.es.store :refer [def-es-store]]
+            [ctim.schemas.asset :as asset-schema]
+            [schema-tools.core :as st]
+            [ctia.http.routes.crud :refer [entity-crud-routes]]
+            [ctia.http.routes.common :as routes.common]
+            [flanders.utils :as fu]
+            [schema.core :as s]))
+
+(def-acl-schema Asset
+  asset-schema/Asset
+  "asset")
+
+(def-acl-schema PartialAsset
+  (fu/optionalize-all asset-schema/Asset)
+  "partial-asset")
+
+(s/defschema PartialAssetList
+  [PartialAsset])
+
+(def-acl-schema NewAsset
+  asset-schema/NewAsset
+  "new-asset")
+
+(def-stored-schema StoredAsset Asset)
+
+(s/defschema PartialStoredAsset
+  (csu/optional-keys-schema StoredAsset))
+
+(def realize-asset (default-realize-fn "asset" NewAsset StoredAsset))
+
+(def asset-mapping
+  {"asset"
+   {:dynamic false
+    :properties
+    (merge
+     em/base-entity-mapping
+     em/describable-entity-mapping
+     em/sourcable-entity-mapping
+     em/stored-entity-mapping
+     {:identity   em/tg-identity
```

Comment:
the identity property is not in the [CTIM schema](https://github.com/threatgrid/ctim/blob/master/src/ctim/schemas/asset.cljc#L28).